Ginger/GingerCore/Drivers/ASCF/ASCFDriver.cs

```diff
@@ -987,7 +987,7 @@ object IWindowExplorer.GetElementData(ElementInfo ElementInfo, eLocateBy element
             return null;
         }
 
-        ObservableList<ElementLocator> IWindowExplorer.GetElementLocators(ElementInfo ElementInfo)
+        ObservableList<ElementLocator> IWindowExplorer.GetElementLocators(ElementInfo ElementInfo,ObservableList<ElementLocator> ElementLocatorsSettingsList=null)
```

Comment:
Please use the EnableFirendlyLocator flag defined on ElementLocate class instead of changing all methods